bfver

```diff
@@ -108,6 +108,21 @@ print_bfb_file_vers () {
     # Keep UEFI version and use it to determine BSP version as well
     UEFI_VER="$(strings -el "$BFB_PATH" | grep "BlueField" | cut -d':' -f 2)"
 
+    UEFI_IMAGE="$BFB_PATH"
+    if [ -z "$UEFI_VER" ]; then
+      mkdir "$TMP_DIR"/uefi
+      cd "$TMP_DIR"/uefi
+      mlx-mkbfb -x "$BFB_PATH"
+      gzipped=$(file dump-bl33-v0 | grep gzip)
+      if [ -n "$gzipped" ]; then
+        mv dump-bl33-v0 dump-bl33-v0.gz
+        gunzip dump-bl33-v0.gz
+        UEFI_VER="$(strings -el dump-bl33-v0 | grep "BlueField" | cut -d':' -f 2)"
+      fi
```

Comment:
Might want to put a short error message here if gzip couldn't be detected.

Reply:
done

Reply:
Oh, sorry, I meant checking `gzipped=$(file dump-bl33-v0 | grep gzip)`. If we can't find "gzip" in the `file` output, print an error. Presence of `gzip` can be assumed due to the requirements of the package.

Reply:
Thanks! Done, could you take a look at the new version?